README.md

```diff
@@ -20,7 +20,7 @@ go get github.com/moesif/moesif-aws-lambda-go
 
 Or, if you are already using Go Modules, specify a version number as well:
 ```shell
-go get github.com/moesif/moesif-aws-lambda-go@v1.0.1
```

Comment:
can you also update frontend code..similar to https://github.com/Moesif/analytics-web-portal-react/pull/771

Reply:
or if you plan to make additional changes, then it ok to do at the end i think